package.json

```diff
@@ -43,11 +43,11 @@
     "@ember/test-helpers": "^2.9.4",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@lblod/ember-acmidm-login": "^2.0.0-beta.1",
+    "@lblod/ember-acmidm-login": "^2.1.0",
     "@lblod/ember-environment-banner": "^0.5.0",
-    "@lblod/ember-mock-login": "0.7.0",
-    "@lblod/ember-rdfa-editor": "9.13.0",
-    "@lblod/ember-rdfa-editor-lblod-plugins": "19.3.1",
+    "@lblod/ember-mock-login": "^0.10.0",
+    "@lblod/ember-rdfa-editor": "9.14.0-dev.4448c75b33c1acd4715f63492d02980e7fb0662d",
```

Comment:
Wouldn't it be better to just set it to 9.14.0 and avoid these dev versions? or not sure what they are

Reply:
These are the development builds from the `release-commit` woodpecker pipeline for the editor or plugins. This is necessary to test with an unreleased version of those dependencies without requiring npm linking (or yalc). Given how much of a mess npm link and even yalc are, it's much more sane this way (especially when working on changing dependencies). We just need to make sure to change them for released versions before we release GN.

Reply:
So does this make sense to merge before those versions are released?

Reply:
To me, yes, as it allows us to continue working on this (for example to fix the problems that Elena and I noticed, but want to do in a separate PR), without having to release many editor and plugins versions which may not be release ready as they need more testing. Happy to wait if others prefer though.